### PR TITLE
message 功能进行实现 #26

### PR DESCRIPTION
**Pull request by guo40020**
#26

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,9 @@
     "wechaty-puppet": "^1.11.14"
   },
   "dependencies": {
+    "file-box": "^1.4.12",
+    "memory-card": "^1.0.3",
+    "wechaty-puppet": "^1.10.2",
```
Comment (bung87) [inline package.json:66]: we are depending on `"wechaty-puppet": "^0.41.9"`

**Comment by bung87** [comment]
It seems your PR base on main branch not dev branch, dependencies and code style  both different.

**Comment by guo40020** [comment]
> It seems your PR base on main branch not dev branch, dependencies and code style both different.

what should i do now?

**Comment by bung87** [comment]
you can use some diff merge tool, pick up your code into new branch that based on dev.

**Comment by guo40020** [comment]
> 

will do. Thx

**Pull request closed without merging**